build.js

```diff
@@ -257,11 +257,12 @@ process.nextTick(function () {
       name: 'TypeScript + core-js',
       url: 'https://www.typescriptlang.org/',
       target_file: 'es6/compilers/typescript-core-js.html',
-      polyfills: ["node_modules/core-js/client/core.js"],
+      polyfills: ["node_modules/core-js-bundle/index.js"],
       compiler: function (code) {
         return ts.transpile(code, { downlevelIteration: true });
       }
     },
+    /*
```

Comment:
This should be removed?

Reply:
Because it's required for Closure Compiler tests which now can't be built.

Reply:
That seems like something that needs to be fixed separately. cc @teppeis ?

Reply:
Seems since @teppeis provide data not only for ES6, he has an alternative tests runner.

Reply:
@ljharb @zloirock I'm using my own build scripts for closure compiler: https://github.com/teppeis/closure-compiler-es-compat-table
Because it's too slow to run tests one by one in Java, I use [nailgun](https://github.com/facebook/nailgun) to reduce the overhead of starting JVM and add some convenient features.

So it is not a problem for me to delete this closure part of build.js, but I do not know if it should be deleted for compat-table project.